playbook/dist/playbook.css

```diff
@@ -17075,7 +17075,7 @@ Background colors ------------------*/
   transition: all 0.15s ease; }
   @media (hover: hover) {
     [class^=pb_selectable_icon]:hover * {
-      transition: all 0.15s ease;
+      transition: transform 0.15s ease;
```

Comment:
If you leave this as `all` then the custom icon disappears and it's also a bit sloppy to transition `all` when you only want `transform`.